docs/documentation/usage.rst

```diff
@@ -316,6 +285,113 @@ Several constraint variables can be defined.
 
 Also, see :ref:`get-variable-list`.
 
+***************************
+Advanced configuration file
```

Comment:
I just realized this:
```suggestion
Advanced configuration
```
I think this is important, because I don't want any user to be wondering if there is another "advanced" configuration file out there.